pkg/fakedata/column.go

```diff
@@ -44,7 +45,7 @@ func NewColumns(keys []string) (cols Columns, err error) {
 		}
 
 		if _, ok := generators[key]; !ok {
-			errors.WriteString(fmt.Sprintf("Unknown generator: %s.\n", key))
+			fmt.Fprintf(&errors, "Unknown generator: %s.\n", key)
```

Comment:
@kevingimbel a tiny change to the code we just merged (together the the comment in the same file). I've recently realised that using a buffer  with `fmt.Fprintf`  reads better than using `WriteString` + `fmt.Sprintf`. At least I think it does :)